decidim-assemblies/spec/system/filter_assemblies_spec.rb

```diff
@@ -77,14 +92,12 @@
     let!(:assembly_with_area) { create(:assembly, area:, organization:) }
     let!(:assembly_without_area) { create(:assembly, organization:) }
 
-    before do
-      visit decidim_assemblies.assemblies_path
-    end
-
     context "and choosing an area" do
       before do
         skip "REDESIGN_PENDING - This test fails with redesigned filters using checkboxes. The issue is addressed in https://github.com/decidim/decidim/issues/10801"
 
+        visit decidim_assemblies.assemblies_path
```

Comment:
@entantoencuanto now that the issue is closed, should we update the test?

Reply:
let's enable the the test!

Reply:
The issue has been closed because the behavior that makes the test fail is accepted, when there is only an area in the organization checking the only existing area activates the "all" option which includes assemblies without area.